src/collectd.conf.pod

```diff
@@ -3855,6 +3855,158 @@ B<Note>: There is no need to notify the daemon after moving or removing the
 log file (e.E<nbsp>g. when rotating the logs). The plugin reopens the file
 for each line it writes.
 
+=head2 Plugin C<logparser>
+
+The I<logparser> plugin is used to parse different kinds of logs. Setting proper
+options you can choose strings to collect. Plugin searches the log file for
+messages which contain several matches (two or more). When all mandatory matches
+are found then it sends proper notification containing all fetched values.
+
+B<Synopsis:>
+
+  <Plugin logparser>
+    <Logfile "/var/log/syslog">
+      FirstFullRead false
+      <Message "pcie_errors">
+        DefaultType "pcie_error"
+        DefaultSeverity "warning"
+        <Match "aer error">
+          Regex "AER:.*error received"
+          SubmatchIdx -1
+        </Match>
+        <Match "incident time">
+          Regex "(... .. ..:..:..) .* pcieport.*AER"
+          SubmatchIdx 1
+          IsMandatory false
+        </Match>
+        <Match "root port">
+          Regex "pcieport (.*): AER:"
+          SubmatchIdx 1
+          IsMandatory true
+        </Match>
+        <Match "device">
+          PluginInstance true
+          Regex " ([0-9a-fA-F:\\.]*): PCIe Bus Error"
+          SubmatchIdx 1
+          IsMandatory false
+        </Match>
+        <Match "severity_mandatory">
+          Regex "severity="
+          SubMatchIdx -1
+        </Match>
+        <Match "nonfatal">
+          Regex "severity=.*\\([nN]on-[fF]atal"
+          TypeInstance "non_fatal"
+          IsMandatory false
+        </Match>
+        <Match "fatal">
+          Regex "severity=.*\\([fF]atal"
+          Severity "failure"
+          TypeInstance "fatal"
+          IsMandatory false
+        </Match>
+        <Match "corrected">
+          Regex "severity=Corrected"
+          TypeInstance "correctable"
+          IsMandatory false
+        </Match>
+        <Match "error type">
+          Regex "type=(.*),"
+          SubmatchIdx 1
+          IsMandatory false
+        </Match>
+       <Match "id">
+          Regex ", id=(.*)"
+          SubmatchIdx 1
+        </Match>
+      </Message>
+    </Logfile>
+  </Plugin>
+
+B<Options:>
+
+=over 4
+
+=item B<Logfile> I<File>
+
+The B<Logfile> block defines file to search. It may contain one or more
+B<Message> blocks which are defined below.
+
+=item B<FirstFullRead> I<true>|I<false>
+
+Set to true if want to read the file from the beginning first time.
+
+=item B<Message> I<Name>
+
+B<Message> block contains matches to search the log file for.
+
+=item B<DefaultPluginInstance> I<String>
+
+Sets the default plugin instance.
```

Comment:
Please reword to something like:
`Sets the default content of the plugin instance part of the message.`